src/app/infobox/infobox.component.html

```diff
@@ -0,0 +1,7 @@
+<div *ngIf="this.description" class="card">
+    <div>
+      <h2><b>{{this.title}}</b></h2>
+      <p>{{this.description | slice:0:600}}<a href='https://en.wikipedia.org/wiki/{{this.title}}'>...more</a></p>
```

Comment:
In this case by clicking ```...more```, user will expect more content getting appended below the content in infobox. But here, the actual task of ```...more``` is to migrate to the corresponding wikipedia page. Either remove it and make the heading an active-link-heading which redirects the user to corresponding wikipedia page, or please see if there is any other way around.

Reply:
Is it possible to display logo corresponding to the search query too?
Like in google:

![screenshot from 2018-06-01 12-09-44](https://user-images.githubusercontent.com/16608864/40825329-dec183ca-6594-11e8-803f-0687246bf0fb.png)

Reply:
@simsausaurabh The image on `google.com` is returned by the Google Images API. If we are to use the Wikipedia API doesn't have a specific API for returning the main image, it just returns an array.

We can use the first image, but it doesn't guarantee the most relevant image.

Reply:
I am talking about the image extracted from wikipedia api, see if it works.

>guarantee the most relevant image.

I guess the result image will be for the query only? We'll see if it works.

Reply:
I am currently going through Wikipedia API and if possible( which I think is possible) I will add it in further patches. There is already an open issue for that. https://github.com/fossasia/susper.com/issues/181 
 https://github.com/fossasia/susper.com/issues/904

Reply:
Sure, We cannot display whole content. I am saying we need to redirect user with some better interactive and meaningful interface(not ```more```).